src/hooks/useFieldValidationResults.ts

```diff
@@ -6,9 +6,11 @@ export function useFieldValidationResults(field: FormField) {
   const [warnings, setWarnings] = useState([]);
 
   useEffect(() => {
-    if (field.meta?.submission) {
-      setErrors((prevErrors) => [...prevErrors, ...(field.meta.submission.errors || [])]);
-      setWarnings((prevWarnings) => [...prevWarnings, ...(field.meta.submission.warnings || [])]);
+    if (field.meta?.submission?.errors) {
```

Comment:
The meta object is initialised at a earlier stage so it's not necessary to optionally chain it.